skiko/src/nativeJsMain/cpp/common.h

```diff
@@ -129,8 +129,11 @@ KNativePointer ptrToInterop(T* ptr) {
 #ifdef __clang__
 __attribute__((noreturn))
```

Comment:
Guess would be better to make it like that

```
#ifdef __clang__
#define NO_RETURN __attribute__((noreturn))
#elif defined(_MSC_VER) 
#define NO_RETURN __attribute__((noreturn))
#else
#define NO_RETURN __declspec(noreturn)
#endif
```
and declare functions unconditionally.